caom2utils/caom2utils/tests/data/taos_h5file/20220201T200117/taosii.py

```diff
@@ -0,0 +1,50 @@
+from astropy import units
+from astropy.coordinates import SkyCoord
+
+
+def get_exposure(base):
+    b = base.get('base')
+    mjdrunstart = b['header']['timeseries']['mjdrunstart']
+    mjdrunend = b['header']['timeseries']['mjdrunend']
+    result = 0.0
+    if mjdrunstart is not None and mjdrunend is not None:
+        result = mjdrunend - mjdrunstart
+    return result
+
+
+def get_target_position_cval1(base):
+    ra, dec_ignore = _get_target_position(base)
+    return ra
+
+
+def get_target_position_cval2(base):
+    ra_ignore, dec = _get_target_position(base)
+    return dec
+
+
+def _get_target_position(base):
+    import logging
```

Comment:
These imports can be moved up at the start of the file I think

Reply:
They're vestigial, and shouldn't be there at all. I' still in the "learning HDF5" phase, and trying to figure out errors.